### PR TITLE
Data Sync downstream reivew

### PR DESCRIPTION
**Pull request by jstaffor**
Issues captured by docs[1] and more, for the Data Sync service.

[1] https://docs.google.com/spreadsheets/d/1oWrlhGVFj8ixH9usO10-Db3I3Ss16gx2N_wdiDA4TB8/edit#gid=1366847283

### CODE REVIEW

**Comment by jstaffor** [comment]
@finp - Ready for a sanity check and alas, the inevitable send-off to github nirvana.

**Comment by finp** [comment]
@jstaffor  does the fact that sync-framework-docs has been merged to master (just now) affect this PR?

**Comment by jstaffor** [comment]
@finp - this PR was created against the master branch, after this https://github.com/aerogear/mobile-docs/pull/613 was merged.

File: modules/ROOT/pages/_partials/sync/js-client-advanced-topics.adoc [hunk 1]
```diff
@@ -37,10 +37,12 @@ let config = {
 The {sync-client} uses the link:https://www.npmjs.com/package/debug[debug module] to log debug messages.
 
 To enable debug logs please execute the following code in a browser's console:
-`localStorage.debug = 'AeroGearSync:*'`.
+
+* `localStorage.debug = 'AeroGearSync:*'`
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-advanced-topics.adoc]: avoid single bullets,
and better as a code block

File: modules/ROOT/pages/_partials/sync/js-client-advanced-topics.adoc [hunk 2]
```diff
@@ -37,10 +37,12 @@ let config = {
...
 
 Certain features can be enabled separately:
-`localStorage.debug = 'AeroGearSync:OfflineMutations*'`
+
+* `localStorage.debug = 'AeroGearSync:OfflineMutations*'`
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-advanced-topics.adoc]: as above

File: modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .Prerequisites
-* [x] You have Node.js and npm installed.
-* [x] You have created an empty web project that supports ES6.  The link:https://webpack.js.org/guides/getting-started/[webpack getting started] project is a useful guide.
-* [x] You have completed the server getting started guide and the application is running.
+* You have Node.js and npm installed.
+* You have created an empty web project that supports ES6.  The link:https://webpack.js.org/guides/getting-started/[webpack getting started] project is a useful guide.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-getting-started.adoc]:
```suggestion
* You have created an empty web project that supports ES6, for example, using the link:https://webpack.js.org/guides/getting-started/[webpack getting started] project.
```

File: modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc [hunk 4]
```diff
@@ -63,10 +63,10 @@ exampleQuery(...) @onlineOnly {
 
 In order to get notification about offline related events, the `offlineQueueListener` in the config object should be used. The following events will be emitted:
 
-* `onOperationEnqueued` - Called when new operation is being added to offline queue
-* `onOperationSuccess` - Called when back online and operation succeeds
-* `onOperationFailure` - Called when back online and operation fails with GraphQL error
-* `queueCleared` - Called when offline operation queue is cleared
+* `onOperationEnqueued` - Called when new operation is being added to offline queue.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc]: not sentence, better without the full stops

File: modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc [hunk 5]
```diff
@@ -108,8 +108,8 @@ To resolve conflicts on the client side, a `conflictStrategy` needs to be provid
 
 To implement a custom conflict resolution strategy provide at least one of the parameters below.
 
-* strategies - a dictionary object where each key is the name of a mutation and the value is the custom action to take for a conflict caused by that mutation
-* default - the default behavior to use if one of your mutations is not listed in `strategies`
+* strategies - a dictionary object where each key is the name of a mutation and the value is the custom action to take for a conflict caused by that mutation.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc]: as above

File: modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc [hunk 6]
```diff
@@ -6,10 +6,10 @@ To do this GraphQL Subscriptions utilise websockets to enable clients to subscri
...
 The architecture of websockets is as follows:
 
-* Client connects to websocket server
-* Upon certain events, the server can publish the results of these events to the websocket
-* Any _currently connected_ client to that websocket will receive these results
-* The client can close the connection at any time and will no longer recieve updates
+* Client connects to websocket server.
+* Upon certain events, the server can publish the results of these events to the websocket.
+* Any _currently connected_ client to that websocket will receive these results.
+* The client can close the connection at any time and will no longer recieve updates.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc]:
```suggestion
* The client can close the connection at any time and no longer receives updates.
```
also, a few of 'will's in prev lines

**Pull request merged**